examples/rivets/readme.md

```diff
@@ -0,0 +1,15 @@
+# Rivets TodoMVC Example
+
+> Rivets.js is a lightweight data binding and templating system that facilitates building data-driven views. It is agnostic about every aspect of a front-end MV(C|VM|P) stack, making it easy to introduce it into your current workflow or to use it as part of your own custom front-end stack comprised of other libraries.
+
+## Learning Rivets
+
+The [Rivets homepage](http://rivetsjs.com) is a great resource for getting started.
+
+Here are some links you may find helpful:
+
+* [Guide](http://rivetsjs.com/docs/guide/)
+* [Binder Reference](http://rivetsjs.com/docs/reference/)
+* [Rivets on Stack Overflow](http://stackoverflow.com/questions/tagged/rivets.js)
```

Comment:
Stack overflow and linking up the official site twice is okay but isn't there anything else we can bring up in here? Interesting blogposts maybe?